.env.test

```diff
@@ -0,0 +1,16 @@
+DEBUG=true
+SECRET_KEY='tests'
+DATABASE_URL='postgresql://postgres@localhost/opencraft'
+HUEY_ALWAYS_EAGER=true
+OPENSTACK_USER='test'
+OPENSTACK_PASSWORD='pass'
+OPENSTACK_TENANT='test-tenant'
+OPENSTACK_AUTH_URL='http://example.com/auth/url'
+OPENSTACK_REGION='test-region'
```

Comment:
@dachary Good point - and to switch to the client API, setting these would be necessary anyway.